packages/TorneloScoresheet/src/components/DragAndDrop/DragAndDropContext/DragAndDropContext.tsx

```diff
@@ -87,6 +90,67 @@ export const useHitTest = (
   };
 };
 
+type ClickToMoveContextState = {
+  fromSquare: unknown | null;
+  toSquare: unknown | null;
+};
+const ClickToMoveContext = React.createContext<
+  [
+    ClickToMoveContextState,
+    React.Dispatch<React.SetStateAction<ClickToMoveContextState>>,
+  ]
```

Comment:
also all this clikc to move stuff should be in its own file instead of drag and drop context's file.